crates/nu-command/src/math/arctan2.rs

```diff
@@ -0,0 +1,117 @@
+use nu_engine::CallExt;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, Example, PipelineData, ShellError, Signature, Span, SyntaxShape, Type, Value,
+};
+
+#[derive(Clone)]
+pub struct SubCommand;
+
+impl Command for SubCommand {
+    fn name(&self) -> &str {
+        "math arctan2"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("math arctan2")
+            .switch("degrees", "Return degrees instead of radians", Some('d'))
+            .input_output_types(vec![(Type::Number, Type::Float)])
+            .vectorizes_over_list(true)
```

Comment:
From a user perspective I feel like the current convention for the math commands to also work on list breaks down for `math arctan2`. 

Typically you have tuples of `y` and `x` you want to operate over. Following the general pattern of the other `math` commands, you could broadcast over the `y` operand while the `x` would always be fixed. This feels a bit surprising. On the other hand I am not sure if lists with two elements are a better input.

Maybe this is better served with two positional arguments instead but that might also be a bit confusing due to the break from the convention used for the unary functions.